lib/spack/spack/test/cmd/config.py

```diff
@@ -2,18 +2,43 @@
 # Spack Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (Apache-2.0 OR MIT)
-
+import hashlib
 import os
 
-from llnl.util.filesystem import mkdirp
+import pytest
 
 import spack.config
 import spack.environment as ev
+import spack.util.spack_yaml as syaml
+from llnl.util.filesystem import mkdirp
 from spack.main import SpackCommand
 
 config = SpackCommand('config')
 
 
+def md5sum(file):
```

Comment:
Could this move to a utility class (e.g. in `spack.util.` or `llnl.util.filesystem`)? Also it might be worth creating a higher level method like `files_are_equal(f1, f2)` that calculates the md5 sums internally.